rollup/dependencies.bzl

```diff
@@ -0,0 +1,45 @@
+"Fetch dependencies needed by users."
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
+
+# WARNING: any changes in this function may be BREAKING CHANGES for users
```

Comment:
do you mean any new additions?

Reply:
yeah that's true. This string is everywhere since cloning from an early version of the template :(

Reply:
neeeeeeed bzlmod